jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -556,7 +557,7 @@ public static String decodePath(String path, int offset, int length)
         {
             if (LOG.isDebugEnabled())
                 LOG.debug("{} {}", path.substring(offset, offset + length), e.toString());
-            return decodeISO88591Path(path, offset, length);
+            throw e;
```

Comment:
The NotUtf8Exception should never be thrown now because we hard code REPLACE?

Reply:
We could just remove the ISO8891 fallback here and not change the Utf8Builder..... ah but whilst if feels like we were previously hard coded to throw, this catch made us not really throw, but instead REPLACE_WITH_8859_1, which is not standard and silly anyway.   So this is not such a big change after all.